docker-compose-mysql-rabbitmq.yml

```diff
@@ -81,7 +80,7 @@ services:
     deploy:
       resources:
         limits:
-          memory: 500m
+          memory: 1000m
```

Comment:
Interesting 🤔  https://www.percona.com/blog/mysql-8-and-mysql-5-7-memory-consumption-on-small-devices/
> When moving to MySQL 8 in a development environment, keep in mind it will require more memory than MySQL 5.7 with the same settings

Reply:
nice find 👍 this was exactly what I saw locally